iscsi/targetd/kubernetes/iscsi-provisioner-d.yaml

```diff
@@ -50,12 +50,11 @@ spec:
           imagePullPolicy: Always
           image: quay.io/external_storage/iscsi-controller:latest
           args:
+            - "-v=2"
```

Comment:
I guess that you want to output messages of `glog.V(2).Infoln`. But if so, you need to specify `logtostderr` as well. And I am thinking that you should  set it by default like other provisioners.

https://github.com/kubernetes-incubator/external-storage/blob/79fd34dc25e05705f08690671cde74c04b1bb5ec/ceph/rbd/cmd/rbd-provisioner/main.go#L44

https://github.com/kubernetes-incubator/external-storage/blob/79fd34dc25e05705f08690671cde74c04b1bb5ec/gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go#L784

Reply:
@nak3 I am not sure I get this comment. probably my lack of understanding of `glog`.
is `glog.V(2).Infoln` not going to log to `stdout`? I don't want to log to `stderr`.
Also what is this flag: `flag.Set("logtostderr", "true") ` going to to?

Reply:
@nak3 I am not sure I get this comment. probably my lack of understanding of glog.
is glog.V(2).Infoln not going to log to stdout? I don't want to log to stderr.
Also what is this flag: flag.Set("logtostderr", "true") going to to?

Reply:
Yes we need to add `flag.Set("logtostderr", "true") ` otherwise glog will just write to files by default. Anyway, the most important thing is that the logs show up upon 'kubectl logs _' or 'oc logs _' and we know it works when we set logtostderr to true

`-logtostderr=false
	Logs are written to standard error instead of to files.`

I will merge this and add the flag, I don't expect you to revisit this given how long it's been